frontend/packages/container-security/src/components/summary.tsx

```diff
@@ -46,7 +46,8 @@ export const SecurityBreakdownPopup: React.FC<SecurityBreakdownPopupProps> = (pr
   return (
     <>
       <div className="co-overview-status__control-plane-description">
-        Quay analyzes container images to identify vulnerabilities.
+        Container images from Quay are analyzed to identify vulnerabilities. Images from other
+        registries will not be scanned.
```

Comment:
Grammar nit to make verb parallel with first sentence.

```suggestion
        registries are not scanned.
```